info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -141,6 +157,9 @@ public void queryCivicInformation_succeedWithMockResponse() throws Exception {
     when(httpClient.execute(anyObject(), argumentCaptor.capture())).thenReturn(ELECTION_RESPONSE);
     JsonObject json =
         InfoCompiler.requestHttpAndBuildJsonResponse(httpClient, httpGet);
+        //The response given by api for testing is missing a property
+        JsonObject mockElection = (JsonObject) json.getAsJsonArray("elections").get(0);
+        mockElection.addProperty("ocdDivisionId", "ocd-division/country:us/state:ny");
```

Comment:
`mockElection` is not being tested after creation

Reply:
It is getting tested in the next line as a part of the json
assertThat(json).isEqualTo(electionJson);

Reply:
Oh that's right! Sorry, I missed that